common/app/routes/challenges/components/video/Video.jsx

```diff
@@ -58,13 +61,14 @@ export class Video extends React.Component {
   render() {
     const {
       title,
+      blockName,
       shouldShowQuestions
     } = this.props;
-
+    let titleString = challengeTitle({ blockName, title });
```

Comment:
This shouldn't be generated on every render. This logic should go in a selector.